### PR TITLE
feat(jaas) add config for displaying participants stats and conferenc…

### PR DESCRIPTION
**Pull request by quitrk**
…e subject

### CODE REVIEW

File: react/features/conference/components/native/NavigationBar.js [hunk 1]
```diff
@@ -99,7 +99,8 @@ function _mapStateToProps(state) {
         _conferenceTimerEnabled:
             getFeatureFlag(state, CONFERENCE_TIMER_ENABLED, true) && !state['features/base/config'].hideConferenceTimer,
         _meetingName: getConferenceName(state),
-        _meetingNameEnabled: getFeatureFlag(state, MEETING_NAME_ENABLED, true),
+        _meetingNameEnabled:
+            getFeatureFlag(state, MEETING_NAME_ENABLED, true) && !state['features/base/config'].hideConferenceSubject,
```
Comment (saghul) [inline react/features/conference/components/native/NavigationBar.js:103]: can you please extract all variables from the same state at the top?

File: react/features/conference/components/web/Subject.js [hunk 2]
```diff
@@ -81,7 +87,8 @@ function _mapStateToProps(state) {
 
     return {
         _hideConferenceTimer: Boolean(state['features/base/config'].hideConferenceTimer),
-        _showParticipantCount: participantCount > 2,
+        _showParticipantCount: participantCount > 2 && !state['features/base/config'].hideParticipantsStats,
+        _showSubject: !state['features/base/config'].hideConferenceSubject,
```
Comment (saghul) [inline react/features/conference/components/web/Subject.js:91]: ditto

**Review by saghul — APPROVED** [review]
LGTM with a couple of suggestions.

File: react/features/conference/components/native/NavigationBar.js (hunk 1 above)
Comment (quitrk) [reply react/features/conference/components/native/NavigationBar.js:103]: good call, done

File: react/features/conference/components/web/Subject.js (hunk 2 above)
Comment (quitrk) [reply react/features/conference/components/web/Subject.js]: ditto

**Review by saghul — APPROVED** [review]
Thanks for walking the extra mile!

**Pull request merged**